packages/flutter_tools/flutter_analysis_options

```diff
@@ -28,25 +30,25 @@ analyzer:
 
```

Comment:
I'd rephrase the comments above. "we" made sense when this applied to the code base of the people who wrote the file, but not so much now